plugins/catalog/api-report.md

```diff
@@ -98,275 +104,304 @@ const catalogPlugin: BackstagePlugin<
-//
 // @public (undocumented)
-export const EntityAboutCard: AboutCard;
+export const EntityDependsOnResourcesCard: (
+  props: DependsOnResourcesCardProps,
+) => JSX.Element;
 
-// Warning: (ae-missing-release-tag) "EntityDependencyOfComponentsCard" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-//
 // @public (undocumented)
-export const EntityDependencyOfComponentsCard: ({
-  variant,
-  title,
-}: {
-  variant?: 'gridItem' | undefined;
-  title?: string | undefined;
-}) => JSX.Element;
+export const EntityHasComponentsCard: (
+  props: HasComponentsCardProps,
+) => JSX.Element;
 
-// Warning: (ae-missing-release-tag) "EntityDependsOnComponentsCard" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-//
 // @public (undocumented)
-export const EntityDependsOnComponentsCard: ({
-  variant,
-  title,
-}: {
-  variant?: 'gridItem' | undefined;
-  title?: string | undefined;
-}) => JSX.Element;
+export const EntityHasResourcesCard: (
+  props: HasResourcesCardProps,
+) => JSX.Element;
 
-// Warning: (ae-missing-release-tag) "EntityDependsOnResourcesCard" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-//
 // @public (undocumented)
-export const EntityDependsOnResourcesCard: ({
-  variant,
-}: {
-  variant?: 'gridItem' | undefined;
-}) => JSX.Element;
+export const EntityHasSubcomponentsCard: (
+  props: HasSubcomponentsCardProps,
+) => JSX.Element;
 
-// Warning: (ae-missing-release-tag) "EntityHasComponentsCard" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-//
 // @public (undocumented)
-export const EntityHasComponentsCard: ({
-  variant,
-}: {
-  variant?: 'gridItem' | undefined;
-}) => JSX.Element;
+export const EntityHasSystemsCard: (props: HasSystemsCardProps) => JSX.Element;
 
-// Warning: (ae-missing-release-tag) "EntityHasResourcesCard" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-//
-// @public (undocumented)
-export const EntityHasResourcesCard: ({
-  variant,
-}: {
-  variant?: 'gridItem' | undefined;
-}) => JSX.Element;
+// @public
+export const EntityLayout: {
+  (props: EntityLayoutProps): JSX.Element;
+  Route: (props: EntityLayoutRouteProps) => null;
+};
 
-// Warning: (ae-missing-release-tag) "EntityHasSubcomponentsCard" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-//
 // @public (undocumented)
-export const EntityHasSubcomponentsCard: ({
-  variant,
-}: {
-  variant?: 'gridItem' | undefined;
-}) => JSX.Element;
+export interface EntityLayoutProps {
+  // (undocumented)
+  children?: React_2.ReactNode;
+  // Warning: (ae-forgotten-export) The symbol "contextMenuOptions" needs to be exported by the entry point index.d.ts
```

Comment:
some warnings here, but handle those later right?

Reply:
yeah indeed - that's why i also didn't add it to the list in the api report generator script